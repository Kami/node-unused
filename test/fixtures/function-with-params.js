// function with two unused params
function foo(a, b) {
};

foo();
