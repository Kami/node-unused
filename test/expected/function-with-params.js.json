[
  {
    "name": "a",
    "loc": {
      "line": 2,
      "column": 13
    },
    "param": true
  },
  {
    "name": "b",
    "loc": {
      "line": 2,
      "column": 16
    },
    "param": true
  }
]